src/tests/mocks.tsx

```diff
@@ -48,6 +50,8 @@ export function createMocks(): Mocks {
   Object.assign(global, popupWindowGlobals)
 
   const teardown = () => {
+    // Render a blank div into the app-container before removing globals to trigger any cleanup from the React components themselves
+    ReactDOM.render(<div />, popupWindow.document.getElementById('app-container'))
```

Comment:
Interesting. I don't understand why we need to do this to trigger a cleanup.

Reply:
emoji-mart had some cleanup steps that rely on there being a global window, so we want those to run before we remove the global window